src/Voice.ts

```diff
@@ -1,30 +1,42 @@
 import { Aflr } from "./Aflr";
 import { isInitializedError, isSubmoduleAlreadyInitializedError } from "./Errors";
 import { RequestBase } from "./RequestBase";
-import { IConfig } from "./types";
+import { IConfig, IVoiceFilteringBody } from "./types";
 
 export class VoiceClass {
   #initialized = false;
   #RequestClass!: RequestBase;
   #url = "";
+  #url_parameters = "";
```

Comment:
the convention I was following was `resourcename_url` so it would be great if you could change this to `parameters_url`

Reply:
:+1: